src/components/typography/Display.tsx

```diff
@@ -0,0 +1,28 @@
+import { Text, StyleSheet } from "react-native";
+
+import { global } from "@/constants/DesignTokens";
+
+type DisplayProps = { size?: "display1" | "display2" };
+
+type TextProps = DisplayProps & Text["props"];
+
+export default function Display(props: TextProps) {
+  const { style, size = "display1", ...otherProps } = props;
+
+  return <Text style={[styles.display, styles[size], style]} {...otherProps} />;
+}
```

Comment:
Why are you doing it like that if you can destructure the same way inside the props?
```suggestion
export default function Display({ style, size = "display1", ...otherProps }: TextProps) {
  return <Text style={[styles.display, styles[size], style]} {...otherProps} />;
}
```

Reply:
Because in the expo starter project it is done this way, so I decided to keep it like this.